python/ray/serve/controller.py

```diff
@@ -80,10 +78,11 @@ async def __init__(self,
 
         self.goal_manager = AsyncGoalManager()
         self.http_state = HTTPState(controller_name, detached, http_config)
-        self.endpoint_state = EndpointState(self.kv_store, self.long_poll_host)
-        self.backend_state = BackendState(controller_name, detached,
-                                          self.kv_store, self.long_poll_host,
-                                          self.goal_manager)
+        self.endpoint_state_manager = EndpointStateManager(
+            self.kv_store, self.long_poll_host)
+        self.backend_state_manager = BackendStateManager(
+            controller_name, detached, self.kv_store, self.long_poll_host,
+            self.goal_manager)
```

Comment:
these renames are a reasonable change but please try to do such changes in separate PRs to make it easier on reviewers!